docs/frost-page-title.md

```diff
@@ -19,10 +19,12 @@ The simplest way to use the frost-page-title `addon` is to add __ember-frost-pag
 // config/environment.js
 ...
 	APP: {
-      'frost-page-title-default': 'My App'
-      'frost-page-title-delimiter': '🍺'
+    frostPageTitle: {
```

Comment:
My preference leans towards `frost-page-title` which is the convention seen most often in the Ember ecosystem (`ember-cli-mirage`, `ember-cli-deploy`, `ember-simple-auth`, etc, as well as our own https://github.com/ciena-frost/ember-frost-core/blob/master/docs/frost-icons.md#addon-icon-packs).  Which do you prefer @juwara0 ?

Reply:
I like frost-page-title, but as it's own namespace

Reply:
Oh sorry, yes, I agree.  I just wasn't clear.

```
{
  'frost-page-title': {
    default: '',
    delimiter: ''
}
```

my example doesn't use `title-` as a preface which is cleaner, but not sure if `default` makes sense then - maybe `title`?

We're on the same page so whatever you roll with.

Reply:
Yeah - I think that stays `defaultTitle`. I don't honestly care any which way - but it is nice to get it into a namespace, as we may have other options to add later.